backend/app/usecase/requester/verifier_fake.go

```diff
@@ -0,0 +1,14 @@
+package requester
+
+// VerifierFake is a stub of Verifier to be used in development only.
+type VerifierFake struct {}
+
+// IsHuman checks whether the request is sent by a human user.
```

Comment:
I can probably change this comment to be more reflective of its fake nature.

Reply:
Other fake method comments in the codebase don't seem to be changed so I'm gonna leave it.